dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/enrollments/EnrollmentsTests.java

```diff
@@ -113,19 +113,18 @@ public void shouldImportTeisWithEnrollments(String file) {
         .body("status", Matchers.equalTo("OK"))
         .body("stats.created", equalTo(2));
 
-    response.validateTeis().body("stats.created", Matchers.equalTo(1));
-
+    response.validateTrackedEntities().body("stats.created", Matchers.equalTo(1));
     response.validateEnrollments().body("stats.created", Matchers.equalTo(1));
 
-    // assert that the tei was imported
-    String teiId = response.extractImportedTeis().get(0);
+    // assert that the TE was imported
+    String teId = response.extractImportedTrackedEntities().get(0);
```

Comment:
should it not be `teUid`?

Reply:
I don't want to update this now. There are many vars in this and other of our e2e tests that use the suffix `id` for uids. I also know that this is common in platform as the only id that is exposed is a uid. We can change this in https://dhis2.atlassian.net/browse/DHIS2-16565.

My goal is to remove as much old tracker code as safely and fast as I can. Maybe I should not have gotten into the renaming here 😅